src/main/features/response/views/evidence.njk

```diff
@@ -22,20 +26,46 @@
     <select name="rows[{{ index }}][type]" id="rows[{{ index }}][type]"
       class="form-control-select {% if errorType %}form-control-error{% endif %}" >
 
-      <option value="">-- Please make a selection --</option>
+      <option value="">{{ t('-- Please make a selection --') }}</option>
       {% for row in allEvidenceTypes %}
         <option value="{{ row.value }}" {% if typeValue == row.value %}selected{% endif %}>{{ row.displayValue }}</option>
       {% endfor %}
     </select>
-    <div class="panel panel-border-narrow">
-      {{ textArea(
-            label = 'Describe this evidence in more detail (optional).',
-            name = 'rows[' + index + '][description]',
-            form = form,
-            inputClass='form-control timeline-column-text',
-            rows=2
-         )
-      }}
+    <div class="panel panel-border-narrow {% if not typeValue %} js-hidden {% endif %}">
+      {% set name = 'rows[' + index + '][description]' %}
+      {% set valueDescription = form.valueFor(name) %}
```

Comment:
That and declaration below overrides declarations of the same name made few lines above. We should avoid overriding variables since that is error prone.

These are overridden:
```
{% set errorType = form.errorFor('rows[' + index + '][type]').length > 0 %}
{% set errorDescription = form.errorFor('rows[' + index + '][description]').length > 0 %}
```

Another thing is that from the names I would expect to have non boolean values while these are boolean. Probably we could name these better like `typeErrorExist` if we need these variables.